recipes/scikit-surprise/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scikit-surprise" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "538ff0adc237ac399babf70186a0c46aa2af6809e2b08d2938404291d04be7cf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy 1.11.*
+  run:
+    - setuptools
+    - python
+    - numpy >=1.11
+    - joblib
+    - six
+
+test:
+  imports:
+    - surprise
```

Comment:
Can you add some more tests than just the top level import (imports of the modules should be fine)?

Reply:
Sure! I tried to run the full test suite by adding 
    
    -  requires:
        - pytest
    -  commands:
        - pytest

but the tests were not found.. any idea how to run tests with pytest?

I'll add other imports anyway

Reply:
That's because the source is deleted at the test phase. See https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files for how to use test files from source